src/jackdaw/data.clj

```diff
@@ -60,7 +60,7 @@
 ;; written a Datafiable entry for.
 
 (defmacro ^:private defn->data [name & body]
-  (let [tag (:tag (meta (ffirst (filter vector? body))))]
+  (let [tag (->> body (filter vector?) ffirst meta :tag)]
```

Comment:
I find this harder .. is it just me? It's inside out for Lisp.
We made the opposite change a few days ago in another PR, so perhaps we need that style guide ...

Reply:
ah well I guess it's just personal taste, I generally always thread whenever possible, but I kind of agree that it looks better when it's on multiple lines, in which case I think it's way nicer than the standard way (and more Clojure idiomatic I would say).

Feel free to close this PR though, I don't really have strong feelings about this.

Reply:
I prefer the original as well.